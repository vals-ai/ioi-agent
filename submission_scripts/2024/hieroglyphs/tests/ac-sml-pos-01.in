version https://git-lfs.github.com/spec/v1
oid sha256:4dd92d0d45dc41c23de8ae0c3d5401c500e8fde19b6e24ea45cbc5472de3519a
size 40523
