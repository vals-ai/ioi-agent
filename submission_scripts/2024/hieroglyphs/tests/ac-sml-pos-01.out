version https://git-lfs.github.com/spec/v1
oid sha256:ea681676ddf0097162179a68d565a6b44e225402f1e277d64a5e5f5e1b4440db
size 366
